rules/Php74/Rector/FuncCall/HebrevcToNl2brHebrevRector.php

```diff
@@ -65,11 +65,6 @@ public function refactor(Node $node): ?FuncCall
         }
 
         $node->name = new Name('hebrev');
-        return new FuncCall(
-            new Name('nl2br'),
-            [
-                new Arg($node)
-            ]
-        );
+        return new FuncCall(new Name('nl2br'), [new Arg($node)]);
```

Comment:
This change was made automatically by `composer fix-cs`. I can undo it if required.